lib/genome/groupers/interaction_grouper.rb

```diff
@@ -75,6 +75,13 @@ def self.add_attributes
                 end
               end
             end
+            #roll publications up to interaction level
+            interaction_claim.publications.each do |pub|
+              interaction.publications << pub unless interaction.publications.include? pub
+            end
+            DataModel::InteractionAttribute.where(name: 'PMID').destroy_all
```

Comment:
I wonder if this needs to be part of the grouper since the interaction attributes won't get created going forward? I'm not sure what best practices is for data cleanup like this.